src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -179,6 +179,8 @@ private ConfigProperties() {
     public static final String CACHE_JMX_STATS = "cache.jmx.statistics";
     public static final String CACHE_CONFIG_FILE_URI = JPA_CONFIG_PREFIX + "hibernate.javax.cache.uri";
     public static final String CACHE_ANON_CERT_CONTENT_TTL = "candlepin.cache.anon_cert_content_ttl";
+    public static final String CACHE_ANON_CERT_CONTENT_MAX_ENTRIES =
+        "candlepin.cache.anon_cert_content_max_entries";
```

Comment:
since we already have 2 of those, we can put them under an even more common namespace, and reduce the underscores. E.g.:
```
candlepin.cache.anonymous.cert.content.ttl
candlepin.cache.anonymous.cert.content.max_entries
```

Reply:
That makes sense, I like that a lot more. Updated